discord/abc.py

```diff
@@ -124,7 +263,11 @@ def __subclasshook__(cls, C):
                 return NotImplemented
 
             mro = C.__mro__
-            for attr in ('display_name', 'mention', 'name', 'avatar', 'discriminator', 'bot'):
+            attrs = ('display_name', 'mention', 'name', 'avatar', 'discriminator', 'bot',
+                     'avatar_url', 'avatar_url_as', 'color', 'colour', 'default_avatar',
+                     'default_avatar_url', 'is_avatar_animated', 'mentioned_in', 'permissions_in',
+                     'public_flags', 'system')
```

Comment:
This could probably shortened by using `__dict__` for most of these, but I honestly don't see why this check is even useful in the first place.